src/hal/src/device.rs

```diff
@@ -247,7 +247,7 @@ pub trait Device<B: Backend> {
 
     ///
     fn create_buffer_view(
-        &self, &B::Buffer, format::Format, Range<u64>
+        &self, &B::Buffer, Option<format::Format>, Range<u64>
```

Comment:
How does this work with `None`?

Reply:
Yes, uniform/storage texel buffers with Format::Undefined are allowed. (also called untyped buffers).
It's also handled inside the open AMD drivers.